### ISSUE TITLE
can the code convert personal model?

### ISSUE THREAD

**Issue by zhaobaozi** [xxradon/PytorchToCaffe#14]
can the code convert personal model?not VGG and resnet ..
Thanks @xxradon

**Comment by xxradon** [comment]
yes,it can convert your own model.------------------ 原始邮件 ------------------
发件人: "zhaobaozi"<notifications@github.com>
发送时间: 2019年4月25日(星期四) 上午9:54
收件人: "xxradon/PytorchToCaffe"<PytorchToCaffe@noreply.github.com>;
抄送: "WangDong 王东"<wangdong_cn@qq.com>;"Mention"<mention@noreply.github.com>;
主题: [xxradon/PytorchToCaffe] can the code convert personal model? (#14)



can the code convert personal model?not VGG and resnet ..
 Thanks @xxradon
 
&mdash;

**Comment by AshishSardana** [comment]
Hi @xxradon,
I have an hourglass model trained using PyT and have its .pth weights.
How do I convert it to caffe?

Thanks,
Ashish
